cypress/e2e/ui/Settings/Application-Settings/schedule.cy.js

```diff
@@ -1,171 +1,137 @@
+const TIME_ZONE_FIELD_LABEL = 'Time Zone';
+const START_DATE_FIELD_LABEL = 'Starting Date';
+const START_TIME_FIELD_LABEL = 'Starting Time';
+const INITIAL_SCHEDULE_NAME = 'Test-name';
+const EDITED_SCHEDULE_NAME = 'Dummy-name';
+const INITIAL_DESCRIPTION = 'Test description';
+const EDITED_DESCRIPTION = 'Dummy description';
+const ACTION_TYPE_VM_ANALYSIS = 'vm';
+const ACTION_TYPE_TEMPLATE_ANALYSIS = 'miq_template';
+const ACTION_TYPE_HOST_ANALYSIS = 'host';
+const ACTION_TYPE_CONTAINER_ANALYSIS = 'container_image';
+const ACTION_TYPE_CLUSTER_ANALYSIS = 'emscluster';
+const ACTION_TYPE_DATA_STORE_ANALYSIS = 'storage';
+const ACTION_TYPE_VM_COMPILANCE_CHECK = 'vm_check_compliance';
+const ACTION_TYPE_HOST_COMPILANCE_CHECK = 'host_check_compliance';
+const ACTION_TYPE_CONTAINER_COMPILANCE_CHECK =
+  'container_image_check_compliance';
+const ACTION_TYPE_AUTOMATION_TASKS = 'automation_request';
+const TIMER_TYPE_ONCE = 'Once';
+const TIMER_TYPE_HOURLY = 'Hourly';
+const TIMER_TYPE_DAILY = 'Daily';
+const TIMER_TYPE_WEEKLY = 'Weekly';
+const TIMER_TYPE_MONTHLY = 'Monthly';
+const FREQUENCY_TYPE_HOUR = '1 Hour';
+const TIME_ZONE_TYPE_HAWAII = '(GMT-10:00) Hawaii';
+const INITIAL_START_DATE = '06/30/2025';
+const EDITED_START_DATE = '07/21/2025';
+const START_TIME = '11:23';
+
+// Buttons
+const SAVE_BUTTON_TEXT = 'Save';
+const CANCEL_BUTTON_TEXT = 'Cancel';
+const RESET_BUTTON_TEXT = 'Reset';
+
+// Config options
+const CONFIG_TOOLBAR_BUTTON = 'Configuration';
+const ADD_SCHEDULE_CONFIG_OPTION = 'Add a new Schedule';
+const DELETE_SCHEDULE_CONFIG_OPTION = 'Delete this Schedule from the Database';
+const EDIT_SCHEDULE_CONFIG_OPTION = 'Edit this Schedule';
+const DISABLE_SCHEDULE_CONFIG_OPTION = 'Disable this Schedule';
+const ENABLE_SCHEDULE_CONFIG_OPTION = 'Enable this Schedule';
+const QUEUE_SCHEDULE_CONFIG_OPTION = 'Queue up this Schedule to run now';
+
+// Menu options
+const SETTINGS_OPTION = 'Settings';
+const APP_SETTINGS_OPTION = 'Application Settings';
+
+// Flash message text snippets
+const FLASH_MESSAGE_SCHEDULE_QUEUED = 'queued';
+const FLASH_MESSAGE_OPERATION_CANCELED = 'cancel';
+const FLASH_MESSAGE_SCHEDULE_DISABLED = 'disabled';
+const FLASH_MESSAGE_SCHEDULE_ENABLED = 'enabled';
+const FLASH_MESSAGE_SCHEDULE_SAVED = 'saved';
+const FLASH_MESSAGE_RESET_SCHEDULE = 'reset';
+const FLASH_MESSAGE_SCHEDULE_DELETED = 'delete';
+const FLASH_MESSAGE_FAILED_TO_ADD_SCHEDULE = 'failed';
+
+// Browser alert text snippets
+const BROWSER_ALERT_DELETE_CONFIRM_TEXT = 'removed';
+
+function selectConfigMenu(menuOption) {
+  return cy.toolbar(CONFIG_TOOLBAR_BUTTON, menuOption);
 }
 
 function addSchedule() {
-  selectConfigMenu();
+  // Open add schedule form
+  selectConfigMenu(ADD_SCHEDULE_CONFIG_OPTION);
   // Checks if Save button is disabled initially
-  cy.contains(
-    '#main-content .bx--btn-set button[type="submit"]',
-    saveButton
-  ).should('be.disabled');
+  cy.getFormFooterButtonByType(SAVE_BUTTON_TEXT, 'submit').should(
```

Comment:
https://github.com/ManageIQ/manageiq-ui-classic/pull/9598 removed this function and others, please rebase / replace with the new function.